doc/userguide/rules/http-keywords.rst

```diff
@@ -684,6 +684,41 @@ like ``distance``, ``offset``, ``within``, etc.
 The ``nocase`` keyword is not allowed anymore. Keep in mind that you need
 to specify a lowercase pattern.
 
+http.request_header
+-------------------
+
+Match on the name and value of a HTTP request header (HTTP1 or HTTP2).
+
+For HTTP2, name and value get concatenated by ": ", colon and space.
+Each colon in the name or the value should be escaped as a double colon "::" for detection
```

Comment:
I don't understand this escaping logic. The normal way to escape `:` in `content` is `\:`

Reply:
But then, I would also need to escape backslash itself ;-)